src/component/button/button.tsx

```diff
@@ -1,22 +1,37 @@
+import classNames from 'classnames'
 import type { FC } from 'react'
 import './button.scss'
 
 type ButtonProps = {
-  label: string
+  label?: string
   onClick?: (event?: React.MouseEvent<HTMLButtonElement>) => void
+  iconButtonOnly?: JSX.Element
   icons?: {
     startIcon?: JSX.Element
     endIcon?: JSX.Element
   }
   variant?: 'secondary' | 'tertiary'
 }
 
-export const Button: FC<ButtonProps> = ({ label, onClick, icons, variant = 'secondary' }) => (
-  <button className={`okp4-dataverse-portal-button-main ${variant}`} onClick={onClick}>
-    <div className={`okp4-dataverse-portal-content-container ${variant}`}>
-      {icons?.startIcon && icons.startIcon}
-      <p className={`okp4-dataverse-portal-button-label ${variant}`}>{label}</p>
-      {icons?.endIcon && icons.endIcon}
-    </div>
+export const Button: FC<ButtonProps> = ({
+  label,
+  onClick,
+  iconButtonOnly,
+  icons,
+  variant = 'secondary'
+}) => (
+  <button
+    className={classNames(`okp4-dataverse-portal-button-main ${variant}`, {
```

Comment:
Do you mean that I should remove it from where it's positioned? 
Because even if we don't  give the `variant` property, we still get the default value so it won't be undefined int he classname, right? 
Or I'm missing something?